test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenAssociationSpecs.java

```diff
@@ -382,7 +382,7 @@ public HapiApiSpec dissociateHasExpectedSemantics() {
 						basicKeysAndTokens()
```

Comment:
Do we need the changes in this file?

Reply:
These came from the `nft` branch.

Reply:
Yes, but it is not necessary imo and it was changed without a reason